packages/web-components/src/components/va-modal/va-modal.css

```diff
@@ -0,0 +1,201 @@
+@import '../../mixins/buttons.css';
+@import '../../mixins/focusable.css';
+
+:host {
+  display: block;
+}
+
+.va-modal {
+  background: rgba(50, 58, 69, 0.8);
+  content: ' ';
+  display: block;
+  height: 100%;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 6;
+}
+
+.va-modal-inner {
+  background: var(--color-white);
+  box-sizing: border-box;
+  margin: auto;
+  max-width: 40rem;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 100%;
+  cursor: auto;
+}
+
+.va-modal-close {
+  background-color: transparent;
+  color: var(--color-primary);
+  font-size: 2.25rem;
+  padding: 0;
+  position: absolute;
+  margin: 1.6rem;
+  right: 0;
+  top: 0;
+  width: auto;
+  z-index: 9;
+  white-space: nowrap;
+}
+
+.va-modal-body {
+  margin-right: 2rem;
+  overflow-wrap: break-word;
+  padding: 2rem;
+  word-break: break-word;
+  word-wrap: break-word;
+}
+
+h1 {
+  font-family: Bitter, Georgia, Cambria, 'Times New Roman', Times, serif;
+  font-size: 2rem;
+  line-height: 1.3;
+  margin-bottom: 0.5rem;
+  margin-top: 1.5rem;
+}
+
+h1:focus {
+  outline: 2px solid var(--color-gold-light);
+  outline-offset: 2px;
+}
+
+.va-modal-title {
+  margin: 0 0 1.6rem 0;
+}
+
+i::before {
+  -webkit-font-smoothing: antialiased;
+  content: '\F057'; /* fa-times-circle*/
+  display: inline-block;
+  font-family: 'Font Awesome 5 Free';
+  font-style: normal;
+  font-variant: normal;
+  font-weight: 900;
+  line-height: 1;
+  text-rendering: auto;
+}
+
+.alert-actions button {
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
same thing on the browser prefixes

Reply:
If I remove these styles, it'll look slightly different in Storybook when compared to the React component.

Reply:
Are we okay with that?

Reply:
Interesting @bkjohnson I was under the impression that postcss would handle all the browser specific prefixes. Am I wrong on that? Cause if so we should ignore my comments on this

Reply:
The visual difference is minor (it looks slightly more bold without it). Is PostCSS supposed to be handling this in Storybook?

With it:
<img width="448" alt="Screen Shot 2022-03-01 at 12 53 18" src="https://user-images.githubusercontent.com/36863582/156222348-5aab371e-0237-4cc1-bc7f-f162dc98280c.png">

Without it:
<img width="448" alt="Screen Shot 2022-03-01 at 12 52 53" src="https://user-images.githubusercontent.com/36863582/156222415-e725c46b-d519-4a88-8cde-ca755c6ec57f.png">

Reply:
I had seen postcss do similar things but I may have been wrong that it was doing this for us.

Anyway, if we want this kind of font smoothing we may want to put it in that `buttons.css` file that gets imported. Something to keep in mind though, from [MDN's documentation regarding `font-smooth`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth):

>  This feature is non-standard and is not on a standards track.

I'd also like to write components that don't target or favor specific browsers or OSes. If the build process adds things in for us that's one thing but adding support in the source gets messy (IE is an example of this).

Reply:
I'm okay with removing it since the difference is minor. Should we do that?